docker/Dockerfile

```
@@ -0,0 +1,13 @@
+ARG swift_version=5.6
+ARG ubuntu_version=bionic
+ARG base_image=swift:$swift_version-$ubuntu_version
+FROM --platform=linux/x86_64 $base_image
```

Can we not fix the platform here. Let it take the host platform. Otherwise people with ARM machines always need to cross compile.
```suggestion
FROM $base_image
```